extensions/blocks/podcast-player/icons/episode-icons.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const EpisodeIcon = ( { name, title, children } ) => {
+	const id = `jetpack-podcast-player__${ name }-icon`;
+
+	return (
+		<SVG
+			height="24"
+			viewBox="0 0 24 24"
+			width="24"
+			xmlns="http://www.w3.org/2000/svg"
+			role="img"
+			aria-labelledby={ id }
+		>
+			<title id={ id }>{ __( title, 'jetpack' ) }</title>
+			{ children }
+		</SVG>
+	);
+};
+
+export const playing = (
+	<EpisodeIcon name="playing" title="Playing">
```

Comment:
Let's translate that title, and the Error one below.

Reply:
The translation is called from the `EpisodeIcon` component: https://github.com/Automattic/jetpack/blob/16b41b13007b4db77d903d4606a57a3850ad5d7a/extensions/blocks/podcast-player/icons/episode-icons.js#L19

Does that look OK?

Reply:
That won't work I am afraid. `__()` will not accept variables, only strings, so you'll need to have the translation happening here.

Reply:
I think I see what @jeherve meant. Afaik, `__` calls are statically extracted and that's how they end up in the list of strings to be translated (at least that's how we do that in other projects in A8C). 

Because of that reason, `__` always needs to have the string directly as an argument. Not because of functionality (JS would obviously pass the value from variable) but because we use static analysis on code that looks for `__` and extracts string.

tldr: use prop directly in `<title>` (no translation) and translate it where you call `EpisodeIcon`, eg: `title={ __( 'Playing', 'jetpack' ) }`

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/15101/commits/57dcf7cc5454d925c1865c5af2a5b9f3f955d4a2